spec.bs

```diff
@@ -573,12 +612,18 @@ Issue: Elsewhere, surround algorithms in a `<div algorithm>` block to match, and
     add styling for all algorithms per
     [bikeshed/1472](https://github.com/speced/bikeshed/issues/1472).
 
-To <dfn algorithm export>set the context ID for a batching scope</dfn> given
-a [=string=] |contextId| and a [=batching scope=] |batchingScope|:
+To <dfn algorithm export>set the pre-specified report parameters for a batching
+scope</dfn> given a [=pre-specified report parameters=] |params| and a
+[=batching scope=] |batchingScope|:
 
-1. [=Assert=]: |contextId|'s [=string/length=] is not larger than 64.
-1. Let |contextIdMap| be the [=context ID map=].
-1. [=map/Set=] |contextIdMap|[|batchingScope|] to |contextId|.
+1. Let |contextId| be |params|' [=pre-specified report parameters/context ID=].
+1. [=Assert=]: |contextId| is null or |contextId|'s [=string/length=] is not
+    larger than 64.
+1. Let |filteringIdByteSize| be |params|' [=pre-specified report parameters/
+    filtering ID byte size=].
+1. [=Assert=]: |filteringIdByteSize| is an integer in the range [1, 8].
```

Comment:
Probably consider defining implementation-defined values for the valid range of filtering ID byte size.

Reply:
As above